api/overview/azure/batch.md

```diff
@@ -47,10 +87,40 @@ Install-Package Microsoft.Azure.Management.Batch.Fluent
 dotnet add package Microsoft.Azure.Management.Batch.Fluent
 ```
 
+### Example
+
+The following example retrieves the quota for the subscription, creates an account, and regenerates the primary account key.
+
+```csharp
+/*
+using Microsoft.Azure.Management.Batch;
+using Microsoft.Azure.Management.Batch.Models;
+*/
+using (BatchManagementClient batchManagementClient = new BatchManagementClient(new TokenCredentials(accessToken)))
+{
+    batchManagementClient.SubscriptionId = subscriptionId;
+
+    // Get the account quota for the subscription
+    BatchLocationQuota quotaResponse = await batchManagementClient.Location.GetQuotasAsync(location);

```

Comment:
BatchLocationQuota should be BatchLocationQuotaInner based on the currently released NuGet package